docs/pipeline-components-and-applications/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Pipeline Components and Applications"
 date: "2020-03-24"
-sidebar_position: 130
+sidebar_position: 140
```

Comment:
I think I noticed @stanch already mentioned this to you but you can simply insert your new pages in between the existing ones.

Reply:
done